staging/src/k8s.io/kubectl/pkg/cmd/apply/apply_test.go

```diff
@@ -2361,7 +2361,7 @@ metadata:
     applyset.k8s.io/tooling: kubectl/v0.0.0-master+$Format:%H$
   creationTimestamp: null
   labels:
-    applyset.k8s.io/id: placeholder-todo
+    applyset.k8s.io/id: bXlTZXQudGVzdC5TZWNyZXQu
```

Comment:
This value looks pretty reasonable (but is maybe too short?).  I just checked the label conditions and spotted "unless empty, must begin and end with an alphanumeric character ([a-z0-9A-Z]),"  https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/

So we should add a consistent prefix and suffix, or only add them when the first/last character is the one of the two "odd" base64 characters (+/ or -_)

Maybe something like "as-" or even `applyset-`
We could also use base32 if that's easier, but personally I prefer a constant prefix (the suffix I find less useful, but also nobody is going to notice if we put an extra `a` on the end).

Reply:
I agree probably nobody would notice an extra "a" (or any other letter), but in the PR you made to my PR, you used `-1`. That looks like a potential sequence... is that intentional? Maybe another option could be a version suffix like `v1` or something? Because tools are supposed to validate the values of this, we're going to have to dictate this suffix/prefix too I assume.

Reply:
Yes, I was thinking if we have to add something, we might as well add some sort of version identifier.  That could be "-v1", "-v2" (as I think you did), that could be `-a`, `-b` etc if we wanted to make it a little less explicit.  I like `-v1` and as we have to dictate what to do if the last value of the base64 encoding is not alphanumeric, so I suggest we use this for a version marker (even if we never use it)

Reply:
Yep, `v1` is what I went with!